lib/passport-configurator.js

```diff
@@ -473,6 +483,11 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
     case 'saml':
       strategy = new AuthStrategy(_.defaults({
         callbackUrl: callbackURL,
+        cert: options.certPath ? fs.readFileSync(options.certPath, 'utf-8') : null,
```

Comment:
Can we allow both `cert` and `certPath`?

Reply:
Yes, though we should encourage users to use files rather than hard-coding keys/certificates.  It makes it really easy to accidentally commit a private key to SCM.